test/testall.py

```diff
@@ -39,4 +41,5 @@
             ans = f.read()
       
         if not ans.strip() == result.strip():
-            raise Exception(f"Incorrect output on test case: {file_loc}\nOutput: {result}Expected output: {ans}")
\ No newline at end of file
+            raise Exception(f"Incorrect output on test case: {file_loc}\nOutput: {result}Expected output: {ans}")
+print(f"\npassed {ctr}/{ctr} test cases")
```

Comment:
am i tripping or won't this always display `x/x`? also plz capitalize the p in  `passed`

Reply:
yeah it legit wont print if there an error ... also I like "passed"

Reply:
oh true

Reply:
capitalize it to make it consistent